src/Solver/SteadyState/Kinsol.hpp

```diff
@@ -125,17 +127,17 @@ namespace AnalysisManager
       //                             N_Vector rhsQB, void *user_data);
 
     private:
-      void*           solver_;
-      SUNContext      context_;
-      SUNMatrix       JacobianMat_;
-      SUNLinearSolver linearSolver_;
+      void*           solver_{};
+      SUNContext      context_{};
+      SUNMatrix       JacobianMat_{};
+      SUNLinearSolver linearSolver_{};
 
-      N_Vector yy_;    ///< Solution vector
-      N_Vector scale_; ///< Scaling vector
-      N_Vector tag_;   ///< Tags differential variables
-      N_Vector q_;     ///< Integrand vector
+      N_Vector yy_{};    ///< Solution vector
+      N_Vector scale_{}; ///< Scaling vector
+      N_Vector tag_{};   ///< Tags differential variables
+      N_Vector q_{};     ///< Integrand vector
```

Comment:
I don't think there are differential variables to tag here.
```suggestion
      N_Vector scale_{}; ///< Scaling vector
      N_Vector q_{};     ///< Integrand vector
```